platform/pulpcore/app/models/content.py

```diff
@@ -79,24 +79,26 @@ class Content(MasterModel):
     """
     A piece of managed content.
 
-    Attributes:
-
-        natural_key_fields (tuple): Natural key fields.  Must be models.Field subclasses.
-
     Relations:
 
         notes (GenericKeyValueRelation): Arbitrary information stored with the content.
         artifacts (models.ManyToManyField): Artifacts related to Content through ContentArtifact
     """
     TYPE = 'content'
 
-    natural_key_fields = ()
-
     notes = GenericKeyValueRelation(Notes)
     artifacts = models.ManyToManyField(Artifact, through='ContentArtifact')
 
     class Meta:
         verbose_name_plural = 'content'
+        unique_together = ()
+
+    @classmethod
+    def natural_key_fields(cls):
+        """
+        Returns a list of the natural key fields which usually equates to unique_together fields
+        """
+        return [field for fields in cls._meta.unique_together for field in fields]
```

Comment:
Did you consider using?: https://docs.python.org/3/library/itertools.html#itertools.chain.from_iterable
Like:
```
return list(chain.from_iterable(cls._meta.unique_together))
```